t/api/15-search.t

```diff
@@ -99,6 +100,12 @@ subtest 'Job templates' => sub {
         '/data/1' => undef,
         'no additional job template found'
     );
+    $t->get_ok('/api/v1/experimental/search?q=banana', 'search successful');
+    $t->json_is('/error' => undef, 'no errors');
+    $t->json_is(
+        '/data/0' => {occurrence => 'Cool Group', contents => "fancy-example\nVery posh"},
+        'job template was found by using test suite name'
```

Comment:
Shouldn't we be looking at the test suite called `banana` here? Although this might relate to my other comment, if perhaps I'm not clear what the expected result is.

Reply:
For example, users add a test suite to job group without specifying alias, such as

Reply:
For example, users add a test suite to job group without specifying alias, such as
```
scenarios:
  x86_64:
    aaa: 
    - apparmor
```
The job template has no name and description, only test suite id. If users search `apparmor`, they cannot find this job template. With this modification, users can find the job template by search `apparmor`, and do not need to care if it is the job template name or test suite name. This is also answer to the above question.